src/perturbIC.py

```diff
@@ -21,7 +21,7 @@
 args = parser.parse_args()
 
 if args.seed >= 0:
-    rs = RandomState(MT19937(SeedSequence(args.seed)))
+    rs = RandomState(args.seed)
```

Comment:
@MartinDix would it be alright to simplfy the random generator to just the RandomState?